src/scripts/charts-screenshots.ts

```diff
@@ -0,0 +1,100 @@
+import 'dotenv';
+
+import fs from 'fs';
+// import path from 'path';
+import puppeteer, { Page } from 'puppeteer';
+
+import chartUrls from './constants/chart-urls';
+
+// const iPad = puppeteer.devices['iPad landscape'];
+
+function delay(time: number) {
+  return new Promise(resolve => {
+    setTimeout(() => {
+      resolve('success');
+    }, time);
+  });
+}
+
+const FRONTEND_SITE_URL = (
+  process.env.ALLOWED_ORIGINS || 'https://explorer.pastel.network'
+).split(',')[0];
+
+function getFileSize(base64String: string) {
+  const stringLength = base64String.length - 'data:image/png;base64,'.length;
+
+  const sizeInBytes =
+    (4 * Math.ceil(stringLength / 3) * 0.5624896334383812) / 1024;
+  return sizeInBytes;
+}
+
+async function updateChartScreenshots(): Promise<void> {
+  const browser = await puppeteer.launch({ headless: true });
+  const page = await browser.newPage();
+  // await page.emulate(iPad);
+  await page.setViewport({ width: 1200, height: 1000 });
+  let i = 0;
+  const pageLength = chartUrls.length;
+  for (; i < pageLength; i++) {
+    const fileNameSave = `${chartUrls[i].split('/').pop()}.png`;
+
+    await page.goto(`${FRONTEND_SITE_URL}${chartUrls[i]}`);
+    await delay(3000);
+    const [button] = await page.$x(
+      "//button[contains(text(), 'Download PNG')]",
+    );
+    let saved = false;
+    // get fist file
+    // let saved = false;
+    page.on('response', async response => {
+      let url = response.request().url();
+      const contentType = response.headers()['content-type'];
+
+      if (
+        !saved &&
+        contentType &&
+        contentType.endsWith('image/png') &&
+        url.startsWith('data:image/png;base64,')
+      ) {
+        const fileSize = getFileSize(url);
+        console.log({ fileSize });
+        if (fileSize > 15) {
```

Comment:
Why 15? Please move to a constant

Reply:
I updated.
This commit:
https://github.com/pastelnetwork/pastel-explorer-backend/pull/45/commits/f4b9e3a8a8bd58e4c57214464e70d6cab3945610